recipes/pydrift/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "pydrift" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 572a5f5ee3cfde052ee858cc2ed741fbb5c3df9fbcd44a70fe1d208b751a163e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - poetry
+    - pandas
+    - flake8
+    - scikit-learn
+    - missingno
+    - plotly_express
+    - jupyter
+    - typing-extensions
+    - catboost
+    - shap
+    - pre-commit
+    - pytest
+    - coveralls
+  run:
+    - python
+    - poetry
```

Comment:
According to the pip dependencies: 

```
- catboost >=0.23,<0.24
- coveralls >=2.0.0,<3.0.0
- flake8 >=3.8.1,<4.0.0
- jupyter >=1.0.0,<2.0.0
- missingno >=0.4.2,<0.5.0
- pandas >=1.0.3,<2.0.0
- plotly_express >=0.4.1,<0.5.0
- pre-commit >=2.4.0,<3.0.0
- pytest >=5.4.2,<6.0.0
- scikit-learn >=0.23.1,<0.24.0
- shap >=0.35.0,<0.36.0
- sphinx >=3.0.3,<4.0.0
- sphinx_press_theme >=0.5.1,<0.6.0
- typing-extensions >=3.7.4,<4.0.0
```

Reply:
@jan-janssen I removed the comment and also removing all of dependencies from host, adapting the run ones to your message